src/decorators/account.decorator.ts

```diff
@@ -0,0 +1,8 @@
+import { createParamDecorator, ExecutionContext } from '@nestjs/common';
```

Comment:
Formatação
Quando importar uma tipagem usar o prefixo _type_, facilita a identificação.

```suggestion
import { createParamDecorator, type ExecutionContext } from '@nestjs/common';
```